cmd/dtab.go

```diff
@@ -94,6 +94,80 @@ var (
 		},
 	}
 
+	dtabVersionCmd = &cobra.Command{
+		Use:     "version [name]",
+		Short:   "Get a version by name",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			switch len(args) {
+			case 1:
+				ctl, err := getController()
+				if err != nil {
+					return err
+				}
+				name := args[0]
+				vd, err := ctl.Get(name)
+				if err != nil {
+					return err
+				}
+
+				fmt.Println(vd.Version)
+
+				return nil
+
+			default:
+				return errors.New("get requires a name argument")
+			}
+		},
+	}
+
+	dtabSetCmd = &cobra.Command{
+		Use:     "set [name] [prefix] [destination]",
+		Short:   "Update a dtab destination by name",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			switch len(args) {
+			case 3:
+				ctl, err := getController()
+				if err != nil {
+					return err
+				}
+				name := args[0]
+				vd, err := ctl.Get(name)
+				if err != nil {
+					return err
+				}
+
+				prefix := args[1]
+				destination := args[2]
+
+				found := false
+				for _, dtab := range vd.Dtab {
+					if dtab.Prefix == "/" + prefix {
+						dtab.Destination = destination
+						found = true
+					}
+				}
```

Comment:
Cool, thanks for the info!

As far as handling matching multiple Dentries, what if we made it only change the last entry?  Would that be something you'd want to merge in?  If so, I can whip up a commit for that.  Otherwise, we might just keep a fork for our own use-cases for the time being.

So, using your example, starting with this:
```
/k8s => /#/io.l5d.k8s ;
/dns => /#/io.l5d.dns ;
/svc => /k8s ;
/svc => /dns ;
/svc/foo => /k8s/bar ;
```

And running `set default /svc /dst`, would result in:
```
/k8s => /#/io.l5d.k8s ;
/dns => /#/io.l5d.dns ;
/svc => /k8s ;
/svc => /dst ;
/svc/foo => /k8s/bar ;
```